packages/react-renderer-demo/src/editor-demo/docs-components/data-types.md

```diff
@@ -0,0 +1,13 @@
+import RawComponent from '../common/component/raw-component';
+
+## Introduction
+
+You can specify a type of a component by providing `dataType`, which will automatically validates the component value.
+Because almost everything in html inputs is outputed as a string, adding the `dataType` props will also cast the value to given type.
+
+### Available dataTypes
+
+```jsx
+['integer', 'float', 'number', 'bool', 'string']
```

Comment:
boolean

Reply:
we will make this bool because Mozilla parser returns bool (i know we won't be using it). Let me fix it in the mapper.

Reply:
what about to allow both?

Reply:
Nvm it is actually boolean